articles/architecture-scenarios/b2c/branding.md

```diff
@@ -0,0 +1,121 @@
+
+* [Use version control](/universal-login/version-control) to maintain prior versions
+* Deploy changes to your script via [automated deployments](#)
+
+Once modified, you are responsible for maintaining the script powering your Login Page, including updating the version number of the Lock widget used.
+
+::: warning
+Exercise caution regarding the use of third-party JavaScript on your Login Page, since sensitive security-related information often flows through the page.
+:::
+
+The Login Page comes with [multi-language support](/libraries/lock/v11/i18n) provided by default. In our experience, almost all applications need internationalization/localization, and while it is fairly simple to add during the development phases, it can become difficult at a later stage. As such, we recommend, at the very least, accounting for this need if you do not implement either feature during this phase.
+
+#### Custom login pages
+
+For those who want to build and use a fully-customized user interface using technologies like React or Angular.js, the Auth0 JavaScript SDK ([auth0js](/libraries/auth0js)) allows you to add Auth0 functionality to your custom login page.
+
+### Change Password
+
+The Change Password page is used whenever a user takes advantage of the [Password Reset](#) functionality. Like the Login Page, you can customize the Change Password page to reflect your organization's branding. 
+
+::: panel Content Delivery Networks
+If you're using custom Login and/or Change Password pages, we recommend using an external Content Delivery Network (CDN) to help speed up page load and browser rendering time for your pages.
+:::
+
+### Errors
+
+If there are issues with user sign up or login, Auth0 provides error messages that indicate what the problem is under the hood. However, the default messages are somewhat cryptic, especially to the end user, since it will likely be missing context that only you can supply. As such, we recommend customizing your error pages to provide the missing context-specific information directly to your users.
+
+Furthermore, customizing your error pages allows you to display your branding, not Auth0's, as well as provide useful information to your users as to what should be done next. This information might include a link to a FAQ or how to get in touch with your company's support team or help desk.
+
+At the minimum, we recommend that you provide your visitors with:
+
+* Visual indicators like a friendly-name and your logo for display
+* An email address and URL for your organization's Support team
+
+Auth0 does not provide an out-of-the-box user interface for customizing your Error Pages, but you can use the [Tenant Settings endpoint of the Management API](/api/management/v2#!/Tenants/patch_settings) to configure them. Alternatively, if you can create and host your Error Page then you can have Auth0 direct users to that page instead of using the Auth0-hosted option. 
```

Comment:
```suggestion
Out-of-the-box there is no user interface for customizing an Error Page hosted by Auth0, but you can create one using an external editor and then use the [Tenant Settings endpoint of the Management API](/api/management/v2#!/Tenants/patch_settings) to configure it. Alternatively, if you can create and host your Error Page then you can have Auth0 direct users to that page instead of using the Auth0-hosted option. 
```